examples/Producer/Program.cs

```diff
@@ -3,21 +3,24 @@
 using System.Text;
 using System.Text.Json;
 
+MemphisClient? memphisClient = null;
+
 try
 {
     var options = MemphisClientFactory.GetDefaultOptions();
-    options.Host = "aws-us-east-1.cloud.memphis.dev";
-    options.AccountId = int.Parse(Environment.GetEnvironmentVariable("memphis_account_id"));
-    options.Username = "test_user";
-    options.Password = Environment.GetEnvironmentVariable("memphis_pass");
+  
+    options.Host = "<memphis-host>";
+    // options.AccountId = <memphis-accountId>;
```

Comment:
same as in consumer

Reply:
Will resolve this here, and we will discuss in the consumer comment.